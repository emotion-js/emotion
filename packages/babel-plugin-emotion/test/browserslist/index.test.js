import * as babel from 'babel-core'
import plugin from 'babel-plugin-emotion'

describe('babel css', () => {
  describe('inline', () => {
    test('css basic', () => {
      const basic = `
        css\`
        display: flex;
      \``
      const { code } = babel.transform(basic, {
        plugins: [[plugin]],
        filename: __filename,
        babelrc: false
      })
      expect(code).toMatchSnapshot()
    })
    test('css object', () => {
      const basic = `
        css({
          display: 'flex'
        })`
      const { code } = babel.transform(basic, {
        plugins: [[plugin]],
        filename: __filename,
        babelrc: false
      })
      expect(code).toMatchSnapshot()
    })
  })
})
