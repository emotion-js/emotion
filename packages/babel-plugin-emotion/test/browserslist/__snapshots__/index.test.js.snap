// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel css inline css basic 1`] = `
"
/*#__PURE__*/css([], [], function createEmotionStyledRules() {
        return [{
                \\"display\\": \\"flex\\"
        }];
});"
`;

exports[`babel css inline css object 1`] = `
"
css({
  'display': 'flex'
});"
`;
